src/overlays/actors/ovl_Bg_Hidan_Kowarerukabe/z_bg_hidan_kowarerukabe.c

```diff
@@ -65,7 +66,8 @@ static ColliderJntSphInit sJntSphInit = {
 };
 
 void BgHidanKowarerukabe_InitDynaPoly(BgHidanKowarerukabe* this, GlobalContext* globalCtx) {
-    static CollisionHeader* collisionHeaders[] = { 0x0600D800, 0x0600D878, 0x0600D8F8 };
+    static CollisionHeader* collisionHeaders[] = { &gHidanObjectsCol_00D800, &gHidanObjectsCol_00D878,
+                                                   &gHidanObjectsCol_00D8F8 };
```

Comment:
```suggestion
                                                   &gHidanObjectsCol_00D8F8, };
```
Same thing with trailing commas here.